proxy/Makefile.am

```diff
@@ -46,7 +46,7 @@ AM_CPPFLAGS = \
   -I$(top_srcdir)/mgmt \
   -I$(top_srcdir)/mgmt/utils \
   -I$(top_srcdir)/proxy/api/ts \
-  -I$(top_srcdir)/lib \
+  -I$(top_srcdir)/lib
```

Comment:
Why was this changed? It looks like it would break @OPENSSL_INCLUDES@. If that's not needed it should be removed entirely. My test indicates it is not needed.